python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -406,10 +412,10 @@ class IsrTaskLSSTConfig(pipeBase.PipelineTaskConfig,
         doc="Sky background threshold for edge bleed detection.",
         default=5000.,
     )
-    itlEdgeBleedModelConstant = pexConfig.Field(
+    itlEdgeBleedModelCst = pexConfig.Field(
```

Comment:
Why is this renamed?  And to something that is less clear.  Renaming configs is not something that we can do simply; it formally needs deprecation, etc.